tests/components/togrill/conftest.py

```diff
@@ -0,0 +1,94 @@
+"""Common fixtures for the Gardena Bluetooth tests."""
+
+from collections.abc import Callable, Generator
+from unittest.mock import AsyncMock, Mock, patch
+
+import pytest
+from togrill_bluetooth.client import Client
+from togrill_bluetooth.packets import Packet, PacketA0Notify, PacketNotify
+
+from homeassistant.components.togrill.const import CONF_PROBE_COUNT, DOMAIN
+from homeassistant.const import CONF_ADDRESS, CONF_MODEL
+
+from . import TOGRILL_SERVICE_INFO
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_entry():
+    """Create hass config fixture."""
+    return MockConfigEntry(
+        domain=DOMAIN,
+        data={
+            CONF_ADDRESS: TOGRILL_SERVICE_INFO.address,
+            CONF_MODEL: "Pro-05",
+            CONF_PROBE_COUNT: 2,
+        },
+    )
+
+
+@pytest.fixture(scope="module")
+def mock_unload_entry() -> Generator[AsyncMock]:
+    """Override async_unload_entry."""
+    with patch(
+        "homeassistant.components.togrill.async_unload_entry",
+        return_value=True,
+    ) as mock_unload_entry:
+        yield mock_unload_entry
```

Comment:
why do we patch this one?

Reply:
Because we patch load. This is a generic bug in many of our tests at the moment. We patch load to be a nop, but the core system still consider the entry loaded, which will cause it to call unload. The unload of integration accesses runtime_data for example which has never been set due to the patched load.

This lead to exception on teardown (normally only shown in stdout logs)